src/lsptoolshost/server/roslynLanguageServer.ts

```diff
@@ -615,6 +615,13 @@ export class RoslynLanguageServer {
             path.join(razorPath, 'Targets', 'Microsoft.NET.Sdk.Razor.DesignTime.targets')
         );
 
+        const razorExtensionPath =
```

Comment:
I think we should probably use this support instead of a new specific option for the razorExtensionPath
https://github.com/dotnet/vscode-csharp/blob/main/src/lsptoolshost/extensions/builtInComponents.ts#L16

This would move the option into `componentPaths` instead, but the API handles that for you.

Reply:
oh good catch, I completely forgot that